src/less/modules/breadcrumbs.less

```diff
@@ -1,7 +1,9 @@
 .breadcrumb {
-  margin-left: 15px;
-  margin-top: -20px;
+  //TODO move breadcrumbs out into app-wide directive under navbar not within ui-view
```

Comment:
@psuedoSudo We might want to have a chat with @macite about this first as, as awesome as it would be to have this done, other tasks in the product backlog might need precedence over it.